docs/deployment/server_access_logging.md

```diff
@@ -0,0 +1,40 @@
+---
+id: server_access_logging
+title: S3 Server Access Logging
+hide_title: true
+---
+
+# S3 Server Access Logging
+
+To enable [EMS Reporting](../ems_reporting.md), you need to enable [S3 Server access logging.][awslogging] on all protected buckets.
+
+
+### Via AWS Console.
+
+[Enable Server Access Logging for an S3 Bucket?][howtologging]
```

Comment:
```suggestion
[Enable Server Access Logging for an S3 Bucket][howtologging]
```

I don't think the question mark is necessary?

Reply:
I edited the long title of "how do I Enable....." and didn't catch that.